DatadogTrace/Sources/DatadogTracer.swift

```diff
@@ -6,8 +6,9 @@
 
 import Foundation
 import DatadogInternal
+import OpenTelemetryApi
 
-internal class DatadogTracer: OTTracer {
+internal class DatadogTracer: OTTracer, OpenTelemetryApi.Tracer {
```

Comment:
nit/ It can be `internal final class` I believe.

Reply:
sure